pybaseball/statcast_pitcher_spin.py

```diff
@@ -0,0 +1,140 @@
+from pybaseball import statcast_pitcher
+import pandas as pd
+import numpy as np
+
+K = .005153 #Environmental Constant
+SIG_DIG = 4
+
+def statcast_pitcher_spin(start_dt=None, end_dt=None, player_id=None):
+	pitcher_data = statcast_pitcher(start_dt, end_dt, player_id)
+
+	spin_df = pitcher_data[['release_extension', 'vx0', 'vy0', 'vz0', 'ax', 'ay', 'az', 'release_spin_rate']].copy()
+
+	spin_df = find_release_point(spin_df)
```

Comment:
I think it'd be helpful if these calculations were in a separate function, so they'd be easier to test

something like, 

```
spin_df = find_intermediate_values(spin_df)
pitcher_data[['Mx', 'Mz', 'phi', 'theta']] = spin_df[['Mx', 'Mz', 'phi', 'theta']].copy()
return pitcher_data

def find_intermediate_values(spin_df):
    spin_df = find_release_point(spin_df)
    spin_df = find_release_time(spin_df)
    .
    .
    return spin_df
```